### PR TITLE
Hide action buttons on footer from sent tab

### PR DESCRIPTION
**Pull request by MayaUribe**
### Purpose
Hide action buttons on footer from Sent Tab.

_This repo is CI/CD enabled and staging deployment should be available at:
https://<branch_name>.publish.buffer.com :smile:_

### CODE REVIEW

**Comment by MayaUribe** [comment]
Hey @hamstu whenever you have time, would you mind checking this one out? 😄
I would like to merge this one before the changes Ana and I have for our current tasks.

Thank you!

File: packages/shared-components/PostFooter/index.jsx [hunk 1]
```diff
@@ -75,6 +75,7 @@ const PostFooter = ({
   postDetails,
   sent,
   dragging,
+  isSent,
```
Comment (karelm) [inline packages/shared-components/PostFooter/index.jsx]: isSent has been added, and the sent prop already exists, and its mandatory... do we need to remove the sent variable from here as it doesnt look like its being used anywhere?

**Comment by karelm** [comment]
@anafilipadealmeida + @MayaUribe I think that my comment about sent vs isSent likely applies to multiple files, but I've only highlighted it in one place... I think it would be great if that could get cleaned up here, but I think this is all great work, so if you think its still needed / used by something, then ignore my comment

otherwise I think its okay to get shipped 💯

File: packages/shared-components/PostFooter/index.jsx (hunk 1 above)
Comment (MayaUribe) [reply packages/shared-components/PostFooter/index.jsx]: Hey there @karelm! yes, that sounds about right, sent is no longer used so I will clean that up 😃thank you for catching that up!

Comment (MayaUribe) [reply packages/shared-components/PostFooter/index.jsx]: @karelm would you mind taking a quick look so I merge this one out? :)

**Pull request merged**